components/gateway/.gitignore

```diff
@@ -1 +1,2 @@
-/bin
\ No newline at end of file
+vendor
+.idea
```

Comment:
Consider including here also a binary file produced by before-commit.sh script, in case it is not removed.